src/InsertionsClient.Core/Api/Providers/InsertionApi.cs

```diff
@@ -119,9 +109,9 @@ internal InsertionApi(TimeSpan? maxWaitSeconds = null, TimeSpan? maxDownloadSeco
                         SwrFileReader swrFileReader = new SwrFileReader(_maxConcurrentWorkers);
                         SwrFile[] swrFiles = swrFileReader.LoadSwrFiles(propsFilesRootDirectory);
 
-                    PropsVariableDeducer variableDeducer = new PropsVariableDeducer(InsertionConstants.DefaultNugetFeed, accessToken);
-                    bool deduceOperationResult = variableDeducer.DeduceVariableValues(configUpdater, results.UpdatedNuGets,
-                        swrFiles, out List<PropsFileVariableReference> variables, out string outcomeDetails, _maxDownloadDuration);
+                        PropsVariableDeducer variableDeducer = new PropsVariableDeducer(InsertionConstants.DefaultNugetFeed, accessToken);
```

Comment:
the spacing in a couple of lines in this method looks off

Reply:
I tried to auto-reformat, but nothing changes. Anything specific that looks off?